ansible/roles/de-image-janitor/tasks/main.yml

```diff
@@ -9,7 +9,7 @@
     state: "directory"
     owner: "condor"
     group: "condor"
-    mode: 0755
+    mode: 0775
```

Comment:
I updated the permisisons on this folder so that jobs, which are run as `clauncher`, will be able to write to this folder.